backend/core/serializers.py

```
@@ -1241,7 +1241,15 @@ class Meta:
 class FindingWriteSerializer(BaseModelSerializer):
     class Meta:
         model = Finding
-        exclude = ["created_at", "updated_at"]
+        exclude = ["created_at", "updated_at", "folder"]
+
+    def create(self, validated_data):
+        findings_assessment = validated_data.get("findings_assessment")
+        if not findings_assessment:
+            raise ValidationError({"findings_assessment": "mandatory"})
```

Ruff (0.8.2)

`ValidationError` may be undefined, or defined from star imports

(F405)